src/doc/rustc/src/command-line-arguments.md

```diff
@@ -62,11 +62,15 @@ attribute or on the command-line, it will link a dynamic library if available,
 otherwise it will use a static library. If the kind is specified on the
 command-line, it will override the kind specified in a `link` attribute.
 
+Note that if you don't specify the `KIND`, the default linkage modifiers won't be applied.
+For instance, [`+bundled` linkage modifier][bundled-modifier] won't be used in conjunction with `static` by default.
```

Comment:
If there's no `KIND`, then there's no `static`, I'm not sure what this paragraph is trying to clarify.

Reply:
This might be a little clearer?

```
For example, the `bundled` linking modifier's default is `+bundled`, which would not be applied to a static library (where it otherwise would have been).
```